plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -776,6 +777,7 @@ private static GenericRecord createTestRecord(
                 .set("float_array_col", floatArrayColumn)
                 .set("double_array_col", doubleArrayColumn)
                 .set("long_array_col", longArrayColumn)
+                .set("timestamp_col", updatedAtMillis)
```

Comment:
Can we have a different timestamp here ?

Reply:
added a different value